redisvl/index/index.py

```diff
@@ -1003,6 +1018,56 @@ def search(self, *args, **kwargs) -> "Result":
         except Exception as e:
             raise RedisSearchError(f"Unexpected error while searching: {str(e)}") from e
 
+    def _hybrid_search(self, query: HybridQuery, **kwargs) -> List[Dict[str, Any]]:
+        """Perform a hybrid search against the index, combining text and vector search.
+
+        Args:
+            query (HybridQuery): The text+vector search query to be performed, with configurable fusion methods and
+                post-processing.
+            kwargs: Additional arguments to pass to the redis-py hybrid_search method (e.g. timeout).
+
+        Returns:
+            List[Dict[str, Any]]: The search results ordered by combined score unless otherwise specified.
+
+        Notes:
+            Hybrid search is only available in Redis 8.4.0+, and requires redis-py >= 7.1.0.
+
+        See Also:
+            - `FT.HYBRID command documentation <https://redis.io/docs/latest/commands/ft.hybrid>`_
+            - `redis-py hybrid_search documentation <https://redis.readthedocs.io/en/stable/redismodules.html#redis.commands.search.commands.SearchCommands.hybrid_search>`_
+
+        .. code-block:: python
+
+            from redisvl.query import HybridQuery
+
+            hybrid_query = HybridQuery(
+                text="lorem ipsum dolor sit amet",
+                text_field_name="description",
+                vector=[0.1, 0.2, 0.3],
+                vector_field_name="embedding"
+            )
+
+            results = index.query(hybrid_query)
+
+        """
+        index = self._redis_client.ft(self.schema.index.name)
+        self._validate_hybrid_query(query)
+
+        if not hasattr(index, "hybrid_search"):
+            raise ImportError(_HYBRID_SEARCH_ERROR_MESSAGE)
+
+        results = index.hybrid_search(
```

Comment:
```suggestion
        results = index._hybrid_search(
```
missing the underscore?

Reply:
This is actually calling a method defined on the `redis-py` search index, so it doesn't need to change 👍